### DISCUSSION TITLE
Alternative method of referencing usb devices and configs

### DISCUSSION THREAD

**Discussion by csharpen** [hertzg/rtl_433_docker#57] [General]
Hi,

1. Have you considered referencing the device `/dev/bus/usb` directly?  This would obviate the need for finding the specific usb device ID which may change, as well as allow specifying the device serial number directly (if the user has set it in eeprom). This is helpful when leveraging several RTL-SDR devices.
2. Have you considered referencing a volume mapping for rtl_433.conf to live on the host? In my opinion, the config file is more readable than numerous CLI arguments in a complex environment.

For reference, both of these techniques are used on a related-but-unrelated project, and I think it works well: [rtlamr2mqq](https://github.com/allangood/rtlamr2mqtt)

**Comment by hertzg** [comment]
Hello @csharpen, 

> 1. Have you considered referencing the device /dev/bus/usb directly? This would obviate the need for finding the specific usb device ID which may change, as well as allow specifying the device serial number directly (if the user has set it in eeprom). This is helpful when leveraging several RTL-SDR devices.

:100: This was discussed here already and this is very close to my setup. See my comment with setup and downsides at https://github.com/hertzg/rtl_433_docker/discussions/45#discussioncomment-1775778

> 2. Have you considered referencing a volume mapping for rtl_433.conf to live on the host? In my opinion, the config file is more readable than numerous CLI arguments in a complex environment.

This is possible as well with `-c` flag pointing to your mount point location. :raised_hands: It really depends on the user how they want to use it and what their infrastructure looks like. Best we could do here is provide examples in the README, if you want this I'd be happy to review the pr for it :+1: 

I'm not sure what the issue is here but his seems like a discussion so I'll be converting it to such, let's continue there :+1:
